### ISSUE TITLE
Container 'color' no longer visible on tab.

### ISSUE THREAD

**Issue by shadowmage45** [mozilla/multi-account-containers#2103]
- Multi-Account Containers Version: 7.4.0
- Operating System + Version: Windows 10
- Firefox Version:  91.0
- Other installed Add-ons + Version + Enabled/Disabled-Status:
Adblock Plus - free ad blocker	extension	3.11.1	true	{d10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d}
Firefox Color	extension	2.1.7	true	FirefoxColor@mozilla.com
Firefox Multi-Account Containers	extension	7.4.0	true	@testpilot-containers
FoxyProxy Standard	extension	7.5.1	true	foxyproxy@eric.h.jung
Tampermonkey	extension	4.13.6136	true	firefox@tampermonkey.net
Windowed - floating Youtube/every website	extension	28	true	{477dbe5e-1742-4641-a2c3-b6113bb5cf6e}

### Actual behavior
No container 'color' is visible on the tab handle.  In this image below, each of the 'New Tab' tabs are opened in a separate container; none are showing the container color.
![image](https://user-images.githubusercontent.com/1503399/129381833-7abe684b-cf07-47da-a604-8224ea669da4.png)

### Expected behavior
The 'color' assigned to the container was visible on a horizontal bar on the tab handle.

### Steps to reproduce
1. Launch Firefox
2. Open a container tab.
3. Notice that no 'container color' is displayed on the tab.

### Notes
I have attempted disabling all other extensions, tried various themes and profiles.

The feature worked earlier in the day.  Closed Firefox at one point, and relaunched, and feature was then no longer functioning.  Tabs themselves also look different, so likely was a Firefox update.

**Comment by achernyakevich-sc** [comment]
It looks as duplicate of #2102 

Lets join efforts to move vote for #2102. :)

**Comment by shadowmage45** [comment]
Thanks, Tried looking for a previous issue with similar subject, but didn't spot any in the first few pages.  Either I'm blind, or I gotta work on my search utilization.

Have closed this in favor of the existing linked issue.

Cheers :)

**Comment by achernyakevich-sc** [comment]
It looks we created our issues in parallel. But I was some seconds faster. :)

If you friends could vote for #2102 let them know. Maybe it will help to move this topic forward. Because for me it looks as critical that I can't find my tabs related to PROD system at glance but need to open every tab and look to URL field... :(
